resources/public/api/conf/1.0/common/queryParameters.yaml

```diff
@@ -0,0 +1,8 @@
+uses:
```

Comment:
Should we not structure this file the same way `pathParameters.yaml` is structured? i.e. components -> parameters -> source?